rfc_backmatter.md

```diff
@@ -7,12 +7,14 @@ In order to maintain lossless behavior, all arithmetic used in encoding and deco
 
 Furthermore, the possibility of integer overflow can be eliminated by using large enough data types. Choosing a 64-bit signed data type for all arithmetic involving sample values would make sure the possibility for overflow is eliminated, but usually smaller data types are chosen for increased performance, especially in embedded devices. This section provides guidelines for choosing the right data type in each step of encoding and decoding FLAC files.
 
+In this appendix, signed data types are signed two's complement.
+
 ## Determining necessary data type size
 To find the smallest data type size that is guaranteed not to overflow for a certain sequence of arithmetic operations, the combination of values producing the largest possible result should be considered.
 
 If for example two 16-bit signed integers are added, the largest possible result forms if both values are the largest number that can be represented with a 16-bit signed integer. To store the result, an signed integer data type with at least 17 bits is needed. Similarly, when adding 4 of these values, 18 bits are needed, when adding 8, 19 bits are needed etc. In general, the number of bits necessary when adding numbers together is increased by the log base 2 of the number of values rounded up to the nearest integer. So, when adding 18 unknown values stored in 8 bit signed integers, we need a signed integer data type of at least 13 bits to store the result, as the log base 2 of 18 rounded up is 5.
 
-In case of multiplication, the number of bits needed for the result is the size of the first variable plus the size of the second variable, but counting only one sign bit if working with signed data types. If for example a 16-bit signed integer is multiplied by a 16-bit signed integer, the result needs at least 31 bits to store without overflowing.
+In case of multiplication, the number of bits needed for the result is the size of the first variable plus the size of the second variable. If for example a 16-bit signed integer is multiplied by a 16-bit signed integer, the result needs at least 32 bits to store without overflowing. To show this in practice, the largest signed value that can be stored in 4 bit is -8. -8 * -8 is 64, which needs at least 8 bits (signed) to store.
```

Comment:
```suggestion
In case of multiplication, the number of bits needed for the result is the size of the first variable plus the size of the second variable. If for example a 16-bit signed integer is multiplied by a 16-bit signed integer, the result needs at least 32 bits to store without overflowing. To show this in practice, the largest signed value that can be stored in 4 bit is -8. (-8)*(-8) is 64, which needs at least 8 bits (signed) to store.
```